batch-nft-reveal/deploy/02_Deploy_NFTCollection.ts

```diff
@@ -0,0 +1,58 @@
+import { ethers } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { networkConfig } from "../helper-hardhat-config";
+
+const NFT_NAME = process.env.NFT_NAME || "Your Collection Name";
+const NFT_SYMBOL = process.env.NFT_SYMBOL || "ABC";
```

Comment:
I like the default values for the other parameters but shouldn't these 2 environment variables throw error if not passed?